zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterTest.java

```diff
@@ -153,33 +149,4 @@ void shouldDeserializeMetadataOnOpen() {
       assertThat(metadata.getFirstUserTaskKey(TaskImplementation.ZEEBE_USER_TASK)).isEqualTo(5);
     }
   }
-
-  @Nested
-  final class FlushTest {
-    @Test
-    void shouldUpdateMetadataOnFlush() {
```

Comment:
❓ is this test not still relevant and we should still be updating metadata on flushing, or did I misunderstand? Or in other words, why do we delete this test?